### DISCUSSION TITLE
[@types/smooch] No `render` method exported

### DISCUSSION THREAD

**Discussion by rahulssheth** [DefinitelyTyped/DefinitelyTyped#66253] [Issues with a @types package] [answered]
The Smooch dependency which the `@types/smooch` package is based on exposes a
`render` method for embedded mode. This will take an `HTMLElement`. It doesn't seem
to be exported from the `@types/smooch` package.

https://github.com/zendesk/sunshine-conversations-web#embedded-mode

[a comment by an automated account is not shown]

**Comment by jpsorensen** [comment, marked as answer]
Put up a PR at https://github.com/DefinitelyTyped/DefinitelyTyped/pull/66426.

@rahulssheth I haven't been extensively using the smooch library for a while, so if you want to take a look and make sure what I've written there makes sense that would be great! (And if you want to put up PRs yourself to update this library, I'm more than happy to review them)
